dependency-injection-tools/AutoFacImplementationWeb/AutoFacImplementationWeb/Attributes/InjectAttribute.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace AutoFacImplementationWeb.Attributes
+{
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    public class InjectAttribute : Attribute
+    {
+        public InjectAttribute() : base() { }
+    }
```

Comment:
What is this code for? I don't remember seeing it in the article.

Reply:
Sir, it was just for test i have removed the codes